src/screens/Tank/SettleBalance/SettleBalance.js

```diff
@@ -245,7 +241,7 @@ class SettleBalance extends React.Component<Props, State> {
     } = this.props;
     const { txToSettle } = this.state;
     const showSpinner = !isFetched;
-    const formattedFeedData = groupAndSortByDate(availableToSettleTx, 0);
+    const formattedFeedData = groupSectionsByDate(availableToSettleTx, 1);
```

Comment:
Why did 0 change to 1 here?

Reply:
`groupSectionsByDate` second param is the unit multiplier for date field. Its default value is set to 1000, meaning it expects the date field to contain value in seconds (since epoch), then multiplying it by 1000 to get milliseconds, which are feed into `Date` constructor.

In this case however, the date field already contains value in milliseconds, so the multiplier needs to be 1. The previous code contained a check that did no multiplication when unit multiplier equals 0. I've simplified the code, by removing zero check and using unit multiplier value of 1, which should give the same effect.